server/data/licenceApiClient.ts

```diff
@@ -202,4 +204,18 @@ export default class LicenceApiClient extends RestClient {
       { username: user?.username }
     )
   }
+
+  async recordAuditEvent(auditEvent: AuditEvent, user?: User): Promise<void> {
+    await this.put({ path: `/audit/save`, data: auditEvent }, { username: user?.username })
```

Comment:
Should we be passing the username to record audit events? Feels like this is a case for using the system token